be/src/olap/rowset/beta_rowset.cpp

```diff
@@ -132,19 +132,23 @@ OLAPStatus BetaRowset::link_files_to(const FilePathDesc& dir_desc, RowsetId new_
     return OLAP_SUCCESS;
 }
 
-OLAPStatus BetaRowset::copy_files_to(const std::string& dir) {
+OLAPStatus BetaRowset::copy_files_to(const std::string& dir, RowsetId new_rowset_id) {
     for (int i = 0; i < num_segments(); ++i) {
-        FilePathDesc dst_path_desc = segment_file_path(dir, rowset_id(), i);
+        FilePathDesc dst_path_desc = segment_file_path(dir, new_rowset_id, i);
         Status status = Env::Default()->path_exists(dst_path_desc.filepath);
         if (status.ok()) {
-            LOG(WARNING) << "file already exist: " << dst_path_desc.filepath;
-            return OLAP_ERR_FILE_ALREADY_EXIST;
+            LOG(WARNING) << "file already exist, delete it: " << dst_path_desc.filepath;
+            status = Env::Default()->delete_file(dst_path_desc.filepath);
```

Comment:
The previous behavior just return `OLAP_ERR_FILE_ALREADY_EXIST`.
Why you modify this logic and remove dest path here?